Use Scipy 1.13 instead of 1.14 to be compatible with the foss2024a Scipy-bundle
Author: C. Comito (FZJ JSC, Germany)
--- pyproject.toml.orig	2025-09-10 17:39:16
+++ pyproject.toml 2025-09-17 17:11:07
@@ -50,7 +50,7 @@
     "mpi4py>=3.0.0",
     "torch~=2.0,<2.8.0",
     "torchvision~=0.15",
-    "scipy~=1.14",
+    "scipy~=1.13",
 ]
